projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -1234,6 +1234,7 @@ public void createJob(
     if (StringUtils.isNotEmpty(serviceAccountName)) {
       podSpecBuilder.withServiceAccountName(serviceAccountName);
     }
+    podSpecBuilder.withAutomountServiceAccountToken(false);
```

Comment:
Consider adding this property through the k8s-data-job-template rather than the source code.